packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx

```diff
@@ -96,6 +97,7 @@ export interface Parameters {
     withSocialPostPlaceholders?: false | { fetchOembed: FetchOEmbedFn };
     withVideoPlaceholders?: false | { fetchOembed: FetchOEmbedFn };
     withWebBookmarkPlaceholders?: false | { fetchOembed: FetchOEmbedFn };
+    withUnfurlingPastedUrls?: false | { fetchOembed: FetchOEmbedFn };
```

Comment:
Just nitpicking here, but I believe that `withPastedUrlsUnfolding` sounds a bit more correct 🤔

Reply:
I agree to the order. Will change! :ok_hand: 

But do you also insist on changing _unfurling_ to _unfolding_?

Reply:
I'm leaving it as _unfurling_, as this seems to be a common industry term for this functionality

![image](https://github.com/prezly/slate/assets/370680/e7b4008a-8ac9-4c68-af1a-97541cd1584d)

Reply:
> I'm leaving it as unfurling, as this seems to be a common industry term for this functionality

Makes perfect sense!